src/debops/cli.py

```diff
@@ -1,4 +1,3 @@
-# -*- coding: utf-8 -*-
```

Comment:
*slaps himself in the head*

This leaves an empty line in this file, and all others with this fix, doesn't it... Is there a check for that in `codespell` which can be enabled?

Reply:
Not that I know of.

This isn't codespell, it's another hook from the precommit configuration file.